src/components/Loading.js

```diff
@@ -0,0 +1,42 @@
+import { dispatch } from "utils/dispatch";
+import {
+  // defaultMapConfig,
+  TOTAL_NUMBER_OF_MAP_LAYERS,
+} from "utils/constants";
+
+export class Loading {
```

Comment:
maybe a better name might be `LoadingIndicator` to help disambiguate?

Reply:
Definitely good point.